Cargo.toml

```diff
@@ -64,7 +64,7 @@ version = "1"
 [dependencies.snarkvm]
 #path = "../snarkVM"
 git = "https://github.com/AleoHQ/snarkVM.git"
-rev = "f7f5b13"
+branch = "extend-ledger-api"
```

Comment:
Let's stick with `rev`. You can use `rev` from branches.

Note in the `./rust` folder there is a `Cargo.toml` snarkVM `rev` that also needs to be updated